hw/ip/edn/data/edn.hjson

```diff
@@ -379,17 +387,16 @@
                 '''
           resval: "0"
         }
-        { bits: "2",
-          name: "CMD_STS",
+        { bits: "5:2",
+          name: "CMD_TYPE",
           desc: '''
-                This one bit field represents the status code returned with the CSRNG application command ack.
-                It is updated each time a HW command is acknowledged by CSRNG.
-                0b0: Request completed successfully.
-                0b1: Request completed with an error.
+                This field contains the application command type of the hardware controlled command issued last.
+                The application command selects one of five operations to perform.
+                A description of the application command types can be found [here](https://opentitan.org/book/hw/ip/csrng/doc/theory_of_operation.html#command-description).
```

Comment:
Suggest to use relative paths (like `../../csrng/doc/theory_of_operation.md#command-description`) instead, because that will work with local files, and it will work if we change the web interface to the documentation